import os
import subprocess


command = f"python src/graph2vec.py --input-path ../src/subgraphs --output-path ..\src/vectorization.csv"


try:
    os.chdir("..")
    os.chdir("graph2vec-master")
    subprocess.run(command, shell=True, check=True)
    os.chdir("..")
    print("La commande a été exécutée avec succès.")
except subprocess.CalledProcessError as e:
    print(f"Erreur lors de l'exécution de la commande : {e}") 


